main.go

```diff
@@ -71,7 +74,13 @@ func main() {
 	setDefaultConsoleDefinitions(operatorCfg)
 
 	operatorCfg.Operator.SetDefaults()
-
+	zaplogger, err := util.SetupLogging("DEBUG")
+	operatorCfg.Logger = zaplogger
+	if err != nil {
+		fmt.Print("error initiating the logger", err)
+		fmt.Print("Will exit Operator", err)
+		os.Exit(1)
```

Comment:
I don't think we want to exit here.

also can we move the line 78 to post error check.